src/fourierdomain/nuft/nfft_alg.jl

```diff
@@ -107,6 +107,26 @@ end
     return nothing
 end
 
+function EnzymeRules.forward(
+    config::EnzymeRules.FwdConfig,
+    func::Const{typeof(_jlnuft!)}, 
+    ::Type{RT}, 
+    out::Annotation{<:AbstractArray{<:Complex}}, A::Const{<:NFFTPlan},
+    b::Annotation{<:AbstractArray{<:Real}}
+) where RT
+    # Forward rule does not have to return any primal or shadow since the original function returned nothing
```

Comment:
I think we still need to compute the primal since it is needed later on in the computation
```julia
func.val(out.val, A.val, b.val)
```